actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -486,7 +488,11 @@ def picture_tag(*sources, &block)
           if block.present?
             capture(&block).html_safe
           else
-            source_tags = sources.map { |source| tag("source", srcset: resolve_asset_source("image", source, skip_pipeline)) } if sources.size > 1
+            source_tags = sources.map do |source|
+              tag("source",
+               srcset: resolve_asset_source("image", source, skip_pipeline),
+               type: Marcel::MimeType.for(extension: "." + File.extname(source)[1..]))
```

Comment:
```suggestion
               type: Template::Types[File.extname(source)[1..]]&.to_s
```

Instead we can use `Template::Types[extname]` that is already being used in other places, e.g: https://github.com/rails/rails/blob/398b904c3efe00523a626103921f5c8e81ef7b7a/actionview/lib/action_view/helpers/asset_tag_helper.rb#L265 https://github.com/rails/rails/blob/398b904c3efe00523a626103921f5c8e81ef7b7a/actionview/lib/action_view/helpers/asset_tag_helper.rb#L344